Content.Shared/Speech/Components/MeleeSpeechComponent.cs

```diff
@@ -28,6 +42,11 @@ public enum MeleeSpeechUiKey : byte
     Key,
 }
 
+//[Serializable, NetSerializable]
+public sealed class MeleeSpeechConfigureActionEvent : InstantActionEvent
+{
+}
+
```

Comment:
huh?

Reply:
Afaik it's required for the event. Looking at MimePowers, it's put into the System instead, but both the component and system for MimePowers are in Server, where MeleeSpeechComponent is in Shared and System is in Server. I need to be able to access it in Component for the action definition to work.

I just made a bit less obtrusive in Component. Would it be better to put most of Component in Server and just have a few things in a separate SharedComponent?